crates/ty_server/tests/e2e/snapshots/e2e__initialize__open_files_diagnostic_registration_via_initialization.snap

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Should we use inline snapshots for those. They're very small

Reply:
Yeah, good idea, I'll update it